apps/studio/pages/project/[ref]/auth/hooks.tsx

```diff
@@ -15,9 +20,21 @@ const Hooks: NextPageWithLayout = () => {
     return <NoPermission isFullPage resourceText="access your project's auth hooks" />
   } else {
     return (
-      <FormsContainer>
-        <HooksListing />
-      </FormsContainer>
+      <div>
+        <ScaffoldHeader className="pb-0">
+          <ScaffoldContainer id="auth-page-top">
+            <ScaffoldTitle>Auth Hooks</ScaffoldTitle>
+            <ScaffoldDescription>
+              Use Postgres functions or HTTP endpoints to customize the behavior of Supabase Auth to
+              meet your needs.
```

Comment:
remove the period `.` at the end here? other pages do not have `.` at the end of the description